doc/20-HTTP-API.md

```diff
@@ -11,8 +11,12 @@ After creating a source in Icinga Notifications Web,
 the specified credentials can be used via HTTP Basic Authentication to submit a JSON-encoded
 [`Event`](https://github.com/Icinga/icinga-go-library/blob/main/notifications/event/event.go).
 
-The authentication is performed via HTTP Basic Authentication, expecting `source-${id}` as the username,
-`${id}` being the source's `id` within the database, and the configured password.
+The authentication is performed via HTTP Basic Authentication using the source's username and password.
+
+!!! tip
+
+    Before Icinga Notification version 0.2.0, the username was a fixed string based on the source ID, such as `source-${id}`.
+    These names were migrated for release version 0.2.0, but can now be altered within Icinga Notifications Web.
```

Comment:
I think you should rephrase this a bit because to me, it implies as if there was something to migrate from. But there is nothing to migrate for existing sources, the new `username` field simply defaults to the previously used hacky workaround.

Reply:
Rephrased the message to imply that the old username is still valid.